packages/instant/src/containers/connected_zero_ex_instant_container.ts

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
```

Comment:
I don't love adding another layer of indirection to get to the meat of the `ZeroExInstantContainer` here.  What do you think about putting the `_renderPaymentMethodOrBuyOrderProgress` code into its own connected component, instead of creating this `ConnectedZeroExInstantContainer` here? 

 If we did this, I'd be OK with the `_renderPaymentMethodOrBuyOrderProgress` just going into the container

Reply:
I feel like it's ok for this logic to be in here? It seems overkill to create another component for this.

Reply:
I still am not a huge fan of this -- I felt like the existing `ZeroExInstantContainer` was nicer, because the separation of concerns felt more clear and straight forward to me:  ZeroExInstantContainer` was the big parent container, and it is just responsible for setting up the hierarchy of its child components, and any connecting is done within the child components.  

Also, for a new developer, it feels indirect to navigate to the "meat" of the instant markup.  You go `zero_ex_instant` -> `connected_zero_ex_instant_container` -> `zero_ex_instant_container`.

Reply:
This is resolved w/ https://github.com/0xProject/0x-monorepo/pull/1255